src/units/attack_type.hpp

```diff
@@ -99,6 +99,8 @@ class attack_type : public std::enable_shared_from_this<attack_type>
 	std::pair<std::string, std::string> damage_type() const;
 	/** @return A list of alternative_type damage types. */
 	std::set<std::string> alternative_damage_types() const;
+	/** @return A modified damage type or a alternative_type based on highest damage */
```

Comment:
I'm not sure what to suggest yet, but we need better documentation - there's too many similarly-named functions here where neither the name nor the documentation answers the question "which one should I use".

Reply:
documentation changed